### ISSUE TITLE
Tray icon always shows "not connected" even when working

### ISSUE THREAD

**Issue by jaxankey** [xor-gate/syncthing-macosx#18]
Hello!

I installed syncthing-macosx 0.14.8-2 on OSX 10.12.1, and then let the user interface update syncthing to 0.14.11. 

The tray icon always appears as "not connected" (stating "not connected" when hovering the mouse over it too), even when syncthing is working perfectly well, showing all the computers and folders correctly in the web browser. I even have out port forwarding / firewall stuff configured, verified by direct (non-relay) connections to the world outside my lan. 

I also have trouble restarting syncthing from within the UI sometimes, but I think that may be a separate issue.

This bundled program is amazing, by the way. Very glad it's here and so simple!

Cheers,
Jack

**Comment by xor-gate** [comment]
Thanks Jack for reporting, are you able to view the folders in the submenu from the tray? Probably you need to set the API key in the tray preferences.

Connected, not connected is not necessary to have the firewall configured as this is for localhost only (most of the time).

In the next release (no due date yet) the tray application will be able to load the API key from the syncthing XML file.

**Comment by jaxankey** [comment]
Yes I had meant to mention this. I do not see the folders. In preferences
the API key is something totally different from that reported in the web
interface, and it's greyed out so I can't change it.

I'll look forward to the next release!

Cheers,
Jack

**Comment by xor-gate** [comment]
The grayed api key is just a suggestion how it looks like. You need to copy paste from the web-gui into the preferences window. Then hit test en see if it turns green, then the tray config is saved. The downside currently is you need to manual restart the tray :-(

**Comment by jaxankey** [comment]
Actually it's greyed out as in "disabled". I can paste the api key in (not
using ctrl-V, but by right clicking), but cannot push the "save" button,
which is also greyed out, even if I change the localhost address.

**Comment by xor-gate** [comment]
You can only push save when the connection can be establish (first hit test).

**Comment by jaxankey** [comment]
Aha!! Thanks! -Jack

PS- very strange behavior: the text input is greyed out initially, making
it difficult to past the number in (only right-click works), then after the
test, the control is enabled (you can type / paste easily, move the cursor
like normal).

**Comment by xor-gate** [comment]
This sounds like a regression. `cmd + p` behaviour should work as expected... but that is not the case. I have found a solution and it seems to fix the copy-paste behaviour:
http://stackoverflow.com/questions/970707/cocoa-keyboard-shortcuts-in-dialog-without-an-edit-menu

**Comment by xor-gate** [comment]
Related to #20. A fix for cmd+p is committed (not released yet).
